File: app/src/main/java/com/stn/carterminal/network/response/ProvidedService.java
package com.stn.carterminal.network.response;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class ProvidedService implements Serializable {
    @SerializedName("providedServiceId")
    private Long providedServiceId;

    @SerializedName("providedServiceNumber")
    private String providedServiceNumber;

    @SerializedName("vesselName")
    private String vesselName;

    @SerializedName("vesselOwner")
    private String vesselOwner;

    @SerializedName("companyName")
    private String companyName;
}

File: app/src/main/java/com/stn/carterminal/network/response/ProvidedService.kt
package com.stn.carterminal.network.response

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class ProvidedService(
        @SerializedName("providedServiceId") var providedServiceId: Long,
        @SerializedName("providedServiceNumber") var providedServiceNumber: String,
        @SerializedName("vesselName") var vesselName: String,
        @SerializedName("vesselOwner") var vesselOwner: String,
        @SerializedName("companyName") var companyName: String
) : Serializable